### PR TITLE
Created 20 different tasks

### PR DESCRIPTION
**Pull request by jessechong**
We now have 20 different tasks. Each task has two different states to train the ML model...

### CODE REVIEW

File: backend/server/data/tasks.json [hunk 1]
```diff
@@ -39,20 +40,473 @@
...
+                "lonelyness": 2.0,
+                "sleepyness": 0.0,
+                "anxiousness": 3.0,
+                "stress": 0.0,
+                "anger": 0.0
+            }
+        ]
+    },
+
+    "Go to event": { // This would have to retrieve events from the ARC.
```
Comment (kajdreef) [inline backend/server/data/tasks.json]: Good that you mention it, but before we can merge it this comment will have to go. Comments will not parse, and cause errors :)

**Comment by jessechong** [comment]
Alright. Sounds good! Yes, what Matt thought is how I interpreted it as
well.

**Comment by kajdreef** [comment]
If you change that one line we can merge it :+1

**Comment by jessechong** [comment]
all clear. Its merging time!

**Pull request merged**